### ISSUE TITLE
Add option to scaffold web-app from a configuration file

### ISSUE THREAD

**Issue by c12i** [holochain/scaffolding#392]
See [hackmd write up](https://hackmd.io/lCtESDHLSvSHm9zZj_LtRA)

**Comment by Zequez** [comment]
The post you linked requires some sort of permission to read it, even signing up for Hackmd.io still throws error 403. Do you have an alternative link?

**Comment by c12i** [comment]
Yes it does @Zequez, will move over the content here, thanks for the heads up
